templates/certified/model-details.html

```diff
@@ -0,0 +1,263 @@
+        alt="Gateway",
+        width="96",
+        height="100",
+        hi_def=True,
+        loading="auto"
+        ) | safe
+      }}
+      {% endif %}
+      {% if category == "Server SoC" %}
+      {{ image (
+        url="https://assets.ubuntu.com/v1/4e0399a1-chip.svg",
+        alt="Chip",
+        width="84",
+        height="100",
+        hi_def=True,
+        loading="auto"
+        ) | safe
+      }}
+      {% endif %}
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-12">
+      <h2>Release{% if release_details["releases"]|length > 1 %}s{% endif %}</h2>
+      <aside class="p-accordion">
+        <ul class="p-accordion__list">
+          {% for release in release_details["releases"] %}
+          <li class="{% if release_details['releases']|length > 1 %}p-accordion__group{% endif %}">
+            {% if release_details['releases']|length > 1%}
+            <div role="heading" aria-level="3" class="p-accordion__heading">
+              <button type="button" class="p-accordion__tab" style="background-position: top 1.25rem left 1rem;"id="tab{{ loop.index }}" aria-controls="tab{{ loop.index }}-section" aria-expanded="false"><h3 class="u-no-margin--bottom">{{ release.name }}</h3></button>
+            </div>
+            {% else %}
+            <h3>{{ release.name }}</h3>
+            {% endif %}
+            {% if release_details['releases']|length > 1 %}
+            <section class="p-accordion__panel" id="tab{{ loop.index }}-section" aria-hidden="true" aria-labelledby="tab{{ loop.index }}">
+            {% else %}
+            <section>
+            {% endif %}
+            {% if release.level == "Enabled" %}
+              <p class="u-no-margin--bottom">Pre-installed in some regions with a custom Ubuntu image that takes advantage of the system’s hardware features and may include additional software. Standard images of Ubuntu may not work well, or at all.</p>
+            {% elif release.level == "Certified" %}
+              <p>The {{ vendor }} {{ name }} development board with the components described below has been awarded the status of certified for Ubuntu.</p>
+              <p class="u-no-margin--bottom">
+                {% if category == "Desktop" or category == "Laptop" %}
+                <a href="/download/desktop" class="p-button--neutral">Download</a>
+                {% endif %}
+                {% if category == "Ubuntu Core" %}
+                <a href="/download/iot" class="p-button--neutral">Download</a>
+                {% endif %}
+                {% if category == "Server" %}
+                <a href="/download/server" class="p-button--neutral">Download</a>
+                {% endif %}
+                {% if category == "Server SoC" %}
+                <a href="/download/server/arm" class="p-button--neutral">Download</a>
+                {% endif %}
+              </p>
+            {% endif %}
+            {% if release.kernel_version %}
```

Comment:
I think this should be `release_details.kernel` looking at what's being sent to the template from the view? Also should we include the bios too going off what Sheila said?

Reply:
All Done